### ISSUE TITLE
Unable to generate gen 6 random team

### ISSUE THREAD

**Issue by louzhang** [smogon/pokemon-showdown#5888]
When I try to generate a random team using `./pokemon-showdown generate-team gen6randombattle` it returns an error.

Running ubuntu 18.04LTS 64bit
Node v12.10.0

Here's the stack trace:
```
ubuntu@ubuntu:~/Pokemon-Showdown$ ./pokemon-showdown generate-team gen6randombattle
/home/ubuntu/Pokemon-Showdown/data/random-teams.js:1878
                if (pokemon.length < 6) throw new Error(`Could not build a random team for ${this.format} (seed=${seed})`);
                                        ^

Error: Could not build a random team for [Gen 6] Random Battle (seed=18378,31258,48394,31254)
    at RandomGen6Teams.randomTeam (/home/ubuntu/Pokemon-Showdown/data/random-teams.js:1878:33)
    at RandomGen6Teams.getTeam (/home/ubuntu/Pokemon-Showdown/data/random-teams.js:61:45)
    at ModdedDex.generateTeam (/home/ubuntu/Pokemon-Showdown/.sim-dist/dex.js:1052:65)
    at Object.<anonymous> (/home/ubuntu/Pokemon-Showdown/pokemon-showdown:98:33)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:999:10)
    at internal/main/run_main_module.js:17:11
```

**Comment by TheImmortal** [comment]
I wouldn't know why it doesn't generate in a shell. It works fine otherwise.

@Zarel ?

**Comment by Zarel** [comment]
Run it through a debugger?

**Comment by Zarel** [comment]
I _can_ but I'm currently working on client-side Illusion support.

**Comment by Zarel** [comment]
Fixed now, presumably by 43ef4d978548b007668
